doc/syslog/syslog-design.md

```diff
@@ -364,9 +502,52 @@ redis-cli -n 4 HGETALL 'SYSLOG_SERVER|2222::2222'
 4) "514"
 5) "vrf"
 6) "Vrf-Data"
-```
-
-### 2.4.3 Configuration sample
+7) "protocol"
```

Comment:
Please provide configuration example for SYSLOG_CONFIG

Reply:
will add